src/adapter/z-stack/znp/znp.ts

```diff
@@ -325,7 +323,8 @@ class Znp extends events.EventEmitter {
     }
 
     private waitressValidator(zpiObject: ZpiObject, matcher: WaitressMatcher): boolean {
-        const requiredMatch = matcher.type === zpiObject.type && matcher.subsystem == zpiObject.subsystem && matcher.command === zpiObject.command;
+        const requiredMatch =
+            matcher.type === zpiObject.type && matcher.subsystem == zpiObject.subsystem && matcher.command === zpiObject.command.name;
```

Comment:
Should use `zpiObject.command.ID` instead of `name` if possible? Better to compare numbers than strings.

Reply:
That would be better indeed, but would require quite some changes, I'll add it to my list of improvements.